heron/config/src/yaml/conf/local/healthmgr.yaml

```diff
@@ -18,7 +19,11 @@ heron.healthmgr.metricsource.url: http://localhost:8888
 #  health.policy.interval.ms: 120000
 #  BackPressureDetector.noiseFilterMillis: 20
 #  GrowingWaitQueueDetector.limit: 5
+#auto-restart-slow-container:
+#  health.policy.class: com.twitter.heron.healthmgr.policy.AutoRestartSlowContainerPolicy
+#  health.policy.interval.ms: 120000
+#  BackPressureDetector.noiseFilterMillis: 20
 #auto-restart-backpressure-container:
 #  health.policy.class: com.twitter.heron.healthmgr.policy.AutoRestartBackpressureContainerPolicy
 #  health.policy.interval.ms: 120000
-#  BackPressureDetector.noiseFilterMillis: 20
+#  BackPressureDetector.noiseFilterMillis: 60000
```

Comment:
is this expected?

Reply:
This configuration value is just an example showing it may be different from the slow instance configuration. Uses have to tune the value in the field.

Reply:
kk. thx.